### DISCUSSION TITLE
请问下配置订阅的相关

### DISCUSSION THREAD

**Discussion by xenoscccd** [kkkgo/PaoPaoGateWay#47] [问和答] [answered]
我使用的mode=suburl 内网订阅clash配置方式来使用的，但是我发现，我每次更新clash的配置文件的时候，好像并不会同步更新生效，每30S更新一次的规则计算的hash都是不变的，需要重启一下gateway虚拟机才能重新加载生效，请问是我自己的配置错误导致的吗？
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/85212804/9888368a-fc2f-46b3-9076-ca524ac9b92d)

**Comment by kkkgo** [comment, marked as answer]
suburl有单独的订阅时间选项subtime。

**Reply by xenoscccd** [reply]
感谢大佬的回复，刚刚重新看了下详细配置介绍，我的问题，没仔细看。
